samples/meetings-attendance-report/nodejs/assets/sample.json

```diff
@@ -0,0 +1,67 @@
+[
+  {
+    "name": "officedev-microsoft-teams-samples-meetings-attendance-report-nodejs",
+    "source": "officeDev",
+    "title": "Bot sent meeting attendance report in chat.",
+    "shortDescription": "This is a sample application which demonstrates how to get attendance reports using Graph API and send it in meeting chat.",
+    "url": "https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/meetings-attendance-report/nodejs",
```

Comment:
Why do we need this file ?

Reply:
As per Wajeed this would be required in projects.